packages/generator/src/generator.js

```diff
@@ -153,7 +153,7 @@ export default class Generator {
     }
 
     // Render and write the SPA template to the fallback path
-    const { html } = await this.nuxt.renderRoute('/', { spa: true })
+    const { html } = await this.nuxt.server.renderRoute('/', { spa: true })
```

Comment:
Same as above, to me it doesnt sound logical that generator uses the server to renderRoutes. Could it maybe use renderer directly?